storage/databaseremover/customDatabaseRemover.go

```diff
@@ -27,22 +28,32 @@ func NewCustomDatabaseRemover(cfg config.StoragePruningConfig) (*customDatabaseR
 		}, nil
 	}
 
-	shouldRemoveFunc, err := parseCustomPattern(cfg.AccountsTrieSkipRemovalCustomPattern)
+	divisibleEpochs, err := parseCustomPattern(cfg.AccountsTrieSkipRemovalCustomPattern)
 	if err != nil {
 		return nil, fmt.Errorf("%w while creating a custom database remover", err)
 	}
 
 	return &customDatabaseRemover{
-		shouldRemoveHandler: shouldRemoveFunc,
+		divisibleEpochs: divisibleEpochs,
 	}, nil
 }
 
 // ShouldRemove returns the result of the inner handler function
-func (c *customDatabaseRemover) ShouldRemove(dbIdentifier string, epoch uint32) bool {
-	return c.shouldRemoveHandler(dbIdentifier, epoch)
+func (c *customDatabaseRemover) ShouldRemove(_ string, epoch uint32) bool {
```

Comment:
we do not need the first parameter. However it will prove handy if we simply need a new implementation.